allofplos/update.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from . import get_corpus_dir, newarticledir, uncorrected_proofs_text_list
```

Comment:
Why is uncorrect_proofs_text_list in __init__.py? Isn't it specific to the update function? Does anything else use it?

Reply:
because the .txt file is stored in the top-level directory (which made sense when `plos_corpus.py` was there).

Reply:
It will be wiped out when you reinstall allofplos

Reply:
yes, just like people who have installed the corpus directory in the default location. let's make obsolete in future PR